contracts/triggers/AaveTrailingQuotePriceTrigger.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import "../interfaces/ITrigger.sol";
+import "../interfaces/aaveV3/IAaveV3Oracle.sol";
+import "../interfaces/chainlink/IAggregatorV3.sol";
+import "../interfaces/lido/IWStEth.sol";
+import "../auth/AdminAuth.sol";
+import "../DS/DSMath.sol";
+import "../utils/TokenUtils.sol";
+import "../actions/aaveV3/helpers/AaveV3RatioHelper.sol";
+
+/// @title Validates trailing stop for quoted asset price
+contract AaveTrailingQuotePriceTrigger is ITrigger, AdminAuth, DSMath, AaveV3RatioHelper {
+    using TokenUtils for address;
+
+    IAaveV3Oracle public constant aaveOracleV3 =
+        IAaveV3Oracle(AAVE_ORACLE_V3);
+
+    /// @param baseTokenAddr address of the token which is quoted
+    /// @param baseStartRoundId roundId of the base token feed at time of subscription
+    /// @param quoteTokenAddr address of the quote token
+    /// @param quoteTokenAddr roundId of the quote token feed at time of subscription
```

Comment:
natspec incorrect `quoteStartRoundId roundId of the quote token feed at time of subscription`